docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -115,8 +115,9 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
         {
             static void Main(string[] args)
             {
-                var c1 = new MyClass();
-                Console.WriteLine($"Hello World! {c1.ReturnMessage()}");
+                var c1 = new Fluffy-broccolli();
+                            Console.WriteLine("Export-1910190154231820315064419971 (1).zip https://pxsprod01.blob.core.windows.net/6cf49368d7ec1b41d23a43feffffffff/Export-1910190154231820315064419971.zip?sv=2018-03-28&sr=b&sig=rnwuBK6EWeMnV7V0SzzzWiiDzOHrghvodkc92CjWyqA%3D&spr=https&se=2019-10-26T01%3A54%3A53Z&sp=r;
```

Comment:
Indentation is bad here. And Also I think the line is mistakenly splitted into two lines.